srv/salt/_modules/dg.py

```diff
@@ -741,7 +741,7 @@ def _reduce_inventory(disk: dict) -> dict:
         # maybe this can and should be dropped when the fix is public
         if not disk:
             return {}
-        if disk["available"] is True:
+        if disk.get("available", False) is not self._include_unavailable:
```

Comment:
This `if` is hurting my brain, but unless I'm reading it wrong, there's a problem. As written it will do this:

* `disk["available"] is False` and `self._include_unavailable` is False, condition fails, disk is not included (ok)
* `disk["available"] is False` and `self._include_unavailable` is True, condition succeeds, disk is included (ok)
* `disk["available"] is True` and `self._include_unavailable` is False, condition succeeds, disk is included (ok)
* `disk["available"] is True` and `self._include_unavailable` is True, condition fails, disk is not included (not ok)

Maybe `if disk.get("available", False) or self._include_unavailable:` ?

Reply:
good catch, updating